src/SprykerEco/Yves/AmazonPay/Theme/default/components/atoms/amazon-submit-confirmation-button/amazon-submit-confirmation-button.ts

```diff
@@ -0,0 +1,86 @@
+import Component from 'ShopUi/models/component';
+
+declare const window: {
+    OffAmazonPayments: {
+        initConfirmationFlow(sellerId: string, orderId: string, callback: (confirmationFlow: ConfirmationFlow) => void): void
+    }
+};
+
+interface ConfirmationFlow {
+    success(): void;
+    error(): void;
+}
+
+const XHR_SUCCESS_CODE = 200;
+
+export default class AmazonSubmitConfirmationButton extends Component {
```

Comment:
Still bad name. Functional component name should answer on question what does this component. 
`button` in this case just details of implementation.

Reply:
fixed